com.unity.render-pipelines.high-definition/Editor/Material/Decal/DecalProjectorEditor.cs

```diff
@@ -278,7 +280,8 @@ void OnSceneGUI()
 
         void DrawBoxTransformationHandles(DecalProjector decalProjector)
         {
-            using (new Handles.DrawingScope(fullColor, Matrix4x4.TRS(decalProjector.transform.position, decalProjector.transform.rotation, Vector3.one)))
+            Vector3 scale = decalProjector.effectiveScale;
+            using (new Handles.DrawingScope(Color.white, Matrix4x4.TRS(decalProjector.transform.position, decalProjector.transform.rotation, scale)))
```

Comment:
Why the need for color change?

Reply:
Sorry, it's my mistake in porting it from a version of decals without custom gizmo colors. Thanks for noticing.